app/controllers/finance/ecf/change_training_statuses_controller.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+module Finance
+  module ECF
+    class ChangeTrainingStatusesController < BaseController
+      before_action :participant_profile
+
+      def new
+        change_training_status_form
+      end
+
+      def create
+        change_training_status_form.assign_attributes(change_training_status_form_params)
+
+        if change_training_status_form.save
+          set_success_message(heading: "Training status updated successfully.")
+          redirect_to finance_participant_path(participant_profile.user)
+        else
+          render :new
+        end
+      end
+
+    private
+
+      def participant_profile
+        @participant_profile ||= ParticipantProfile.find(params[:participant_profile_id])
+      end
+
+      def change_training_status_form_params
+        return {} unless params.key?(:finance_ecf_change_training_status_form)
```

Comment:
https://guides.rubyonrails.org/action_controller_overview.html#more-examples

you can change this to something like the below to be more concise. the outcome should be the same
```
params.fetch(:finance_ecf_change_training_status_form, {}).permit(
          :training_status,
          :reason,
        )
```